dist/index.html

```diff
@@ -20,30 +26,34 @@
         <nav class="main-header__nav">
           <div class="nav__navigation left">
             <span class="logo">
-              FITNESS
+              <a href="#">FITNE<span class="orange">SS</span></a>
             </span>
-            <ul class="nav-links">
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
+            <ul class="nav-links hide">
+              <li><a href="#">HOME</a></li>
+              <li><a href="#">ABOUT US</a></li>
+              <li><a href="#">PAGE</a></li>
+              <li><a href="#">BLOG</a></li>
+              <li><a href="#">CONTACTS</a></li>
             </ul>
           </div>
-          <div class="nav__login right">
+          <div class="nav__login right hide">
             <ul>
-              <li>Login</li>
-              <li>Sign Up</li>
-              <li><button class="btn">button</button></li>
+              <li><a href="#">LOGIN</a></li>
+              <li><a href="#">SIGN UP</a></li>
+              <li class="btn btn2"><a href="#">CHOOSE PLAN</a></li>
             </ul>
           </div>
+          <div class="show-bar ">
+            <i class="fas fa-bars fa-2x"></i>
+          </div>
         </nav>
+           
       </header>
 
       <div class="main-section__info">
-        <h1>Make Your Body Beautiful</h1>
-        <p>Lorem ipsum dolor sit amet.</p>
-        <button class="btn btn-primary">Book Now</button>
+        <h1>MAKE YOUR BODY <br /><span id="orange">BEAUTIFUL</span></h1>
```

Comment:
[same as this comment](https://github.com/m3ntorship/fitness/pull/11/files#r407260592)